fe/fe-core/src/main/java/com/starrocks/lake/backup/LakeRestoreJob.java

```diff
@@ -125,7 +125,7 @@ protected void prepareAndSendSnapshotTasks(Database db) {
                 Partition part = tbl.getPartition(idChain.getPartId());
                 MaterializedIndex index = part.getIndex(idChain.getIdxId());
                 tablet = (LakeTablet) index.getTablet(idChain.getTabletId());
-                Backend backend = GlobalStateMgr.getCurrentSystemInfo().getBackend(tablet.getPrimaryBackendId());
+                Backend backend = GlobalStateMgr.getCurrentSystemInfo().getBackend(tablet.getPrimaryComputeNodeId());
```

Comment:
There is no guarantee the return value is an instance of `Backend`, maybe an instance of `ComputeNode`